tests/test_eigen_tensor.py

```diff
@@ -100,6 +100,7 @@ def test_round_trip():
     copy.setflags(write=False)
     assert_equal_tensor_ref(m.round_trip_const_view_tensor(copy))
 
+    np.testing.assert_array_equal(tensor_ref[:, ::-1], m.round_trip_tensor(tensor_ref[:, ::-1, :]))
```

Comment:
I'm not sure I understand the logic: don't you want to test whether `a = tensor_ref[:, ::-1, :]` round trips to itself (i.e., `a`)?

Generally, round trip checks (as all other tests) should be ideally performed for both memory layouts not just column major. For reference, [these](https://github.com/sergiud/hogpp/blob/4944e20af0674a6aba4925b05f5c7c8e9aeb6268/tests/python/test_tensor_type_caster.py) are the tests that I wrote for my type casters which you are welcome to reuse (I believe `round_trip_*` functions roughly translate to `pass_` in those tests.)

Reply:
Good point, that was a typo. I fixed this to 

>     np.testing.assert_array_equal(tensor_ref[:, ::-1, :], m.round_trip_tensor(tensor_ref[:, ::-1, :]))

I also implemented your suggestion of having all the tests run with both c and f style.